src/app/core/cache/models/search-param.model.ts

```diff
@@ -0,0 +1,6 @@
+
+export class SearchParam {
```

Comment:
Please add TypeDoc/comments to describe this new class.

Is this class purposefully empty?  It looks a little odd. But, maybe it'll be easier to understand if you add a comment to describe it.